helloworld/elementary_4.c

```diff
@@ -0,0 +1,21 @@
+#include<stdio.h>
+
+int main() {
+
+    int szam;
```

Comment:
Ez a sor is tab-okkal legyen indentálva, mint a többi. Most space-ekkel van.

Reply:
kész